tutorials/math/Legendre.py

```diff
@@ -28,11 +28,14 @@
 L[0].SetMaximum(1)
 L[0].SetMinimum(-1)
 L[0].SetTitle("Legendre polynomials")
-leg.AddEntry(L[0].Draw(), " L_{0}(x)", "l")
-leg.AddEntry(L[1].Draw("same"), " L_{1}(x)", "l")
-leg.AddEntry(L[2].Draw("same"), " L_{2}(x)", "l")
-leg.AddEntry(L[3].Draw("same"), " L_{3}(x)", "l")
-leg.AddEntry(L[4].Draw("same"), " L_{4}(x)", "l")
-leg.Draw()
+
+for idx, val in enumerate(L):
+    leg.AddEntry(val, " L_{%d}(x)"%(idx), "l")
+    if idx == 0:
+        val.Draw()
```

Comment:
I guess it is fine to do the `Draw` after adding the function to the legend, right? Just asking because in the previous version (and in C++) the `Draw` of the functions is done before `AddEntry`.

Reply:
Yes, the result is the same